documentation/tutorials/getting-started-with-ash-json-api.md

```diff
@@ -68,7 +68,7 @@ end
 
 ### Defining routes on the resource
 
-Here we show an example of defining routes on the resource. In general, we suggest
```

Comment:
"In general, we suggest" seemed out of place. It could have been a copy/paste error, or perhaps it just needs a semicolon at the end.
I opted for removing (the "less is more" principle)